src/Examples/IExample.h

```diff
@@ -33,12 +33,30 @@ namespace Examples
         virtual void UpdateCardboardProfile(const float cardboardProfile[]){}
         virtual void NotifyScreenPropertiesChanged(const Eegeo::Rendering::ScreenProperties& screenProperties) = 0;
         
-        // TODO: All examples should be returing a RenderCamera used by example, temporaray hack to make it work with VR example.
-        virtual Eegeo::Camera::RenderCamera& GetRenderCamera(){return m_renderCamera;}
         virtual Eegeo::Camera::CameraState GetCurrentCameraState() const = 0;
-        virtual const Eegeo::VR::VRCameraState& GetVRCameraState(){return m_vrCameraState;};
         virtual void SetVRCameraState(const float headTransform[]){};
-        virtual bool IsVRExample(){return false;};
+        virtual void UpdateWorld(float dt, Eegeo::EegeoWorld& world, Eegeo::Camera::CameraState cameraState, Examples::ScreenPropertiesProvider& screenPropertyProvider, Eegeo::Streaming::IStreamingVolume& streamingVolume)
```

Comment:
This is an interface (as denoted by the leading "I" in the class name). As such, it shouldn't implement any methods.